inference-engine/src/transformations/include/transformations/common_optimizations/hsigmoid_fusion.hpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <utility>
+
+#include <transformations_visibility.hpp>
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API HSigmoidFusion;
+class TRANSFORMATIONS_API HSigmoidFusionWithReluDiv;
```

Comment:
Why do we need to export all matchers if we use only `HSigmoidFusion` class in other libraries?

I propose to export only `HSigmoidFusion`.